test/api/api_test.go

```diff
@@ -190,11 +191,10 @@ func TestUserTierCache(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	body := map[string]string{
-		"email":    email,
-		"password": password,
-	}
-	r, _, err := at.Post("/login", nil, body)
+	bodyParams := url.Values{}
+	bodyParams.Add("email", email)
```

Comment:
Doesn't matter functionality-wise but using `Add` makes me think there is a special reason for using `Add`. Because it's usually used when you need to set multiple values for the same param.

Otherwise `Set` would suffice.

Reply:
Oh, good point. I just read the implementation and `Set` is definitely what I want.